--- ufw-0.33.n/setup.py	2012-08-18 00:12:49.000000000 +0300
+++ ufw-0.33/setup.py	2013-09-16 11:29:34.815967366 +0300
@@ -148,7 +148,7 @@
         if self.root != None:
             confdir = self.root + real_confdir
 
-        defaults = os.path.join(confdir, 'default', 'ufw')
+        defaults = os.path.join(confdir, 'sysconfig', 'ufw')
         ufwconf = os.path.join(confdir, 'ufw', 'ufw.conf')
         sysctl = os.path.join(confdir, 'ufw', 'sysctl.conf')
         before_rules = os.path.join(confdir, 'ufw', 'before.rules')
--- ufw-0.33.n/src/backend.py	2012-08-18 00:12:49.000000000 +0300
+++ ufw-0.33/src/backend.py	2013-09-16 11:29:34.819300854 +0300
@@ -32,8 +32,8 @@
         self.dryrun = dryrun
         self.rules = []
         self.rules6 = []
 
-        self.files = {'defaults': os.path.join(config_dir, 'default/ufw'),
+        self.files = {'defaults': os.path.join(config_dir, 'sysconfig/ufw'),
                       'conf': os.path.join(config_dir, 'ufw/ufw.conf'),
                       'apps': os.path.join(config_dir, 'ufw/applications.d') }
         if extra_files != None:
--- ufw-0.33.n/src/ufw-init-functions	2012-08-18 00:12:49.000000000 +0300
+++ ufw-0.33/src/ufw-init-functions	2013-09-16 11:29:34.819300854 +0300
@@ -20,7 +20,7 @@
 
 PATH="/sbin:/bin:/usr/sbin:/usr/bin"
 
-for s in "#CONFIG_PREFIX#/default/ufw" "#CONFIG_PREFIX#/ufw/ufw.conf" ; do
+for s in "#CONFIG_PREFIX#/sysconfig/ufw" "#CONFIG_PREFIX#/ufw/ufw.conf" ; do
     if [ -s "$s" ]; then
         . "$s"
     else
